JesusFilm/core
# fix: new journey badge

# Description
- Removed redundant custom hook for calling userJourneyOpen mutation
- Moved userJourneyOpen mutation into server side call for pages `/journeys/[journeyId]`, `/journeys/[journeyId]/edit`, `/journeys/[journeyId]/reports`,


[Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/31245472/todos/5864081668)

# How should this PR be QA Tested?
Preconditions
- Before testing the below, you should have no other journeys that is currently giving you a notification badge
- To get a new journey, click the add button on `/` page, and refresh the page before you are automatically redirected
- Visit the other pages via the url using the journey id (you can get this from the journey create mutation)

<br/>

- [ ] New journey badge should disappear when visiting the journey details page
- [ ] New journey badge should disappear when visiting the journey edit page
- [ ] New journey badge should disappear when visiting the journey reports page 

# Checklist
